src/bootstrap_handler.rs

```diff
@@ -184,6 +194,33 @@ mod test {
         path.parent().unwrap().join(name)
     }
 
+    fn create_contacts() -> Contacts {
+        let addr = net::SocketAddrV4::new(net::Ipv4Addr::new(1,2,3,4), 8080);
```

Comment:
I thought we already had helper functions to create random addresses?  If so, I think we should use random addresses.  Ideally we should use random types too, i.e. not limited to TCP and IPv4, but those helpers may not be implemented yet.